morphocut/stream.py

```diff
@@ -15,7 +15,7 @@
 
 class TQDM(Node):
     """
-    Provide a progress indicator via `tqdm`_.
+    Provide a progress indicator via `tqdm`_. Also use the library tqdm
```

Comment:
Let the module docstring of pims inspire yourself ;) -> Small explanation what tqdm is; a hint that tqdm needs to be installed separately, ...

Build the documentation and make sure, everything looks correct: https://trello.com/c/PyWg4rz3/13-build-the-documentation